aspnetcore/tutorials/getting-started-with-swashbuckle/includes/getting-started-with-swashbuckle8.md

```diff
@@ -1,6 +1,9 @@
 
 :::moniker range=">= aspnetcore-6.0 < aspnetcore-9.0"
 
+> [!NOTE]
+> Swashbuckle is not available in .NET 9 and later. For an alternative, see <xref:fundamentals/openapi/overview?view=aspnetcore-9.0&preserve-view=true>.
```

Comment:
@tdykstra  see e32115c991e03c61341bacc7ae8d470c3921a623 which I think is too tricky.